benchmark/config.jl

```diff
@@ -0,0 +1,21 @@
+using Pkg, Distributed, JLD2
+
+const NP = ARGS[1]
+const SOLVER = ARGS[2]
+
+addprocs(parse(Int,NP),exeflags="--project=.")
```

Comment:
Do we need to add explicitly the flag `--project` when using `Distributed.jl`?

Reply:
Yes, according to the [Distributed.jl documentation](https://docs.julialang.org/en/v1/manual/distributed-computing/)